app/static/src/scss/header.scss

```diff
@@ -1,39 +1,40 @@
 @mixin on-mobile {
-    @media screen and (width <= 768px) {
+    @media screen and (width <=768px) {
         @content;
     }
 }
 
 .govuk-header {
-    &__logo {
+    font-family: Arial, sans-serif !important;
+
+    &__logo--ayr {
+
         @include on-mobile {
             padding: 0;
             margin: 0;
         }
     }
 
-    &__link--tna {
+    &__link--ayr {
```

Comment:
(for my understanding) was the --tna suffix just an oversight when it should have always been --ayr since that is the bem pattern we agreed for modifing the govuk styles?

Reply:
I believe the tna suffix was set as it's for "Delivered by The National Archives". Happy to revert back but i've set the main header logo to be under logo--ayr & the sublink to be link--ayr. Both could be reasonable choices

Reply:
your changes makes sense to me 👍